pandas/core/indexes/datetimes.py

```diff
@@ -1770,11 +1770,31 @@ def date(self):
 
     def normalize(self):
         """
-        Return DatetimeIndex with times to midnight. Length is unaltered
+        Return DatetimeIndex with times converted to midnight.
+
+        When using DatetimeIndex, the time can be converted to midnight i.e.
+        00:00:00. This is useful in cases, when the time does not matter in
```

Comment:
This useful in cases where the time does not matter (remove the in DataFrame). The length of the DatetimeIndex is unaltered.

Reply:
maybe mention this does not affect tz's